windows/docs/guides/00-quickstart.md

```diff
@@ -0,0 +1,30 @@
+# Quickstart
+
+Getting started with `robocorp.windows` is as simple as:
+
+```python
+windows.desktop().windows_run("notepad.exe")  # starts Notepad with Windows Run
+notepad = windows.find_window('subname:"Notepad"')  # controls & gets the window into focus
+notepad.find('control:"MenuItemControl" and name:"File"').click()  # clicks the found "File" menu item
```

Comment:
The only thing that looks a little bit annoying, is our helicopter caring for users to use double-quotes even for locators not containing any space inside, so they won't forget to do that when they will really need it.

💡 Although I just tested it and even for locators containing spaces, there's no need anymore to put double-quotes, maybe for legibility and cross-compatibility with `RPA.Windows` only, e.g.:
```python
text_edit = notepad_window.find("name:Text editor")
```
is still working:
<img width="788" alt="Screenshot 2024-02-01 at 3 42 37 PM" src="https://github.com/robocorp/robocorp/assets/709053/41441f1c-8299-4d20-8258-ce140635a9a8">